tailor_distro/generate_bundle_templates.py

```diff
@@ -24,12 +24,12 @@ def get_debian_depends(package: Package):
 
 
 def get_debian_build_depends(package: Package):
-    deps = package.build_depends + package.doc_depends + package.test_depends + package.buildtool_depends
```

Comment:
not sure why we were doing this
- we don't run tests during bundle build
- we run tests against the dev bundle which has all packages
- we always run rosdep before tests in CI which should pull in anything extra.